test/evergreen.yml

```diff
@@ -296,12 +300,11 @@ functions:
           powershell.exe  -NonInteractive '.\test\evergreen\build_windows.ps1' -configure 1 $DEFINED_EVERGREEN_CONFIG_FLAGS ${windows_configure_flags|}
         else
           echo "Using config flags $DEFINED_EVERGREEN_CONFIG_FLAGS ${posix_configure_flags|}"
-          # Fetch the gperftools library if needed. This will also get tcmalloc.
-          if [[ ${ENABLE_TCMALLOC|0} -eq 1 ]] && [ ! -d "./automation-scripts" ]; then
-            is_cmake_build=true
-            git clone git@github.com:wiredtiger/automation-scripts.git
-            . automation-scripts/evergreen/find_gperftools.sh ${s3_access_key} ${s3_secret_key} ${build_variant} $is_cmake_build
+          
+          if [[ ${ENABLE_TCMALLOC|0} -eq 1 ]]; then
```

Comment:
I'm unfamiliar with `${ENABLE_TCMALLOC|0}` construct. Did you mean to use `${ENABLE_TCMALLOC:-0}` ?

Reply:
Hmm.. Now I see this is used multiple times in this file. I have no idea how it works since it doesn't appear like a valid bash expression.

Reply:
It's an evergreen thing I don't understand. Recommend you ask @lukech  @jiechenbo or someone else more familiar with Evergreen. I just put in what I was told. 😄

Reply:
It's an expansion syntax in Evergreen. The curly braces imply it's an Evergreen expansion (instead of a normal shell variable), and the value after "|" is the default value (for the cases expansion value is not overloaded). Check out this Evergreen documentation page for details: https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Configuration-Files#expansions